x/auth/types/codec.go

```diff
@@ -26,6 +26,13 @@ func RegisterCodec(cdc *codec.Codec) {
 	cdc.RegisterConcrete(StdTx{}, "cosmos-sdk/StdTx", nil)
 }
 
+// RegisterAccountTypeCodec registers an external account type defined in
```

Comment:
This PR was to address registering a key type, but this function seems to concern itself with account type?

Reply:
this was the old function that was defined before the changes. I can rename it if you want

Reply:
Well, what are you allowing the caller to do? Register a KeyType or AccountType?